sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs

```diff
@@ -126,31 +124,42 @@ public ServiceBusAdministrationClient(string connectionString)
         /// </summary>
         ///
         /// <param name="fullyQualifiedNamespace">The fully qualified Service Bus namespace to connect to.  This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>.</param>
-        /// <param name="credential">The <see cref="ServiceBusSharedAccessKeyCredential"/> to use for authorization.  Access controls may be specified by the Service Bus namespace or the requested Service Bus entity, depending on Azure configuration.</param>
+        /// <param name="credential">The <see cref="AzureNamedKeyCredential"/> to use for authorization.  Access controls may be specified by the Service Bus namespace or the requested Service Bus entity, depending on Azure configuration.</param>
         /// <param name="options">A set of options to apply when configuring the connection.</param>
         internal ServiceBusAdministrationClient(
             string fullyQualifiedNamespace,
-            ServiceBusSharedAccessKeyCredential credential,
+            AzureNamedKeyCredential credential,
             ServiceBusAdministrationClientOptions options)
+           : this(fullyQualifiedNamespace, TranslateCredential(fullyQualifiedNamespace, credential), options)
         {
-            Argument.AssertWellFormedServiceBusNamespace(fullyQualifiedNamespace, nameof(fullyQualifiedNamespace));
-            Argument.AssertNotNull(credential, nameof(credential));
-
-            options ??= new ServiceBusAdministrationClientOptions();
-            _fullyQualifiedNamespace = fullyQualifiedNamespace;
-
-            var audience = BuildAudienceResource(fullyQualifiedNamespace);
-            var tokenCredential = new ServiceBusTokenCredential(credential.AsSharedAccessSignatureCredential(audience), audience);
+        }
 
-            HttpPipeline pipeline = HttpPipelineBuilder.Build(options);
-            _clientDiagnostics = new ClientDiagnostics(options);
+        /// <summary>
+        /// Initializes a new <see cref="ServiceBusAdministrationClient"/> which can be used to perform administration operations on ServiceBus entities.
+        /// </summary>
+        ///
+        /// <param name="fullyQualifiedNamespace">The fully qualified Service Bus namespace to connect to.  This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>.</param>
+        /// <param name="credential">The <see cref="AzureSasCredential"/> to use for authorization.  Access controls may be specified by the Service Bus namespace or the requested Service Bus entity, depending on Azure configuration.</param>
+        internal ServiceBusAdministrationClient(
```

Comment:
Isn't this meant to be public?

Reply:
Yup.... whoops.   I'll fix that up once the live suite completes.